flowforge-container/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM node:16-alpine
 
 ARG REGISTRY
+ARG REGISTRY_TOKEN
 ARG TAG
+RUN if [[ ! -z "$REGISTRY_TOKEN" ]]; then echo "//npm.pkg.github.com/:_authtoken=$REGISTRY_TOKEN" >> ~/.npmrc ; fi
```

Comment:
should the `npm.pkg.github.com` bit be picked up from `$REGISTRY` or am I misunderstanding what these bits are doing?

Reply:
The problem is that npm wants a full URL for the registry (e.g. `https://npm.pkg.github.com/`) and only a URL fragment (no schema e.g. `//npm.pkg.github.com/` ) for the authentication token key.

I suppose I could just pass the fragment in from the GH Action and then pre-pend the schema in the next line to make it all parametrised.